XRTK-Core/Packages/com.xrtk.core/Editor/Templates~/SystemProfile.txt

```diff
@@ -2,7 +2,9 @@
 
 namespace #NAMESPACE#
 {
+    [CreateAssetMenu(menuName = "#NAMESPACE#/#NAME#/#NAME#Profile", fileName = "#NAME#Profile", order = 99)]
     public class #NAME#Profile : #BASE#
     {
     }
+
```

Comment:
nit extra line

Reply:
You do realise you could have just "done" it.